### ISSUE TITLE
XiitStudio account - I was connect to an admin account for few minutes without reason 

### ISSUE THREAD

**Issue by Hugucool** [itchio/itch.io#722]
I just reloaded my dashboard again and again but PAM , Connect to admin , i thought it's was a joke or a demo and after i was back to my account and now i can't connect to my account !

**Comment by dmu-Lee-W** [comment]
I just had the same thing happen to me, I went to my dashboard to change some settings and I was suddenly apparently logged in to an admin account, leafo. I didn't realise at first, I wasn't paying attention to the account in the top right, I thought that some of the stuff I was seeing were new features until I realised that they were admin features.

And then, just as suddenly, I was logged out and can't login to my regular user account.

I wasn't doing anything out of the ordinary, have no clue how something like this could even happen, and I didn't change anything in the admin.

**Comment by quyse** [comment]
https://twitter.com/itchio/status/918903538952146944

**Comment by leafo** [comment]
Sorry for the trouble everyone. We had development code get deployed by accident. We noticed in about 8 minutes and took it down. We then disabled account access for the next few hours while we reviewed logs to evaluate any potential damage. 

Most users didn't realize what was going on, a few users attempted to make some changes on parts of the site they shouldn't be able to but we've reversed them. 

Account access is back to normal now and we're looking into process changes along with changing account security to ensure this doesn't happen again.

Thanks
